src/App.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import './App.css';
 
+import GoogleLogin from 'react-google-login';
+
+import Login from './Component/Login';
+import Display from './Component/Display';
+
 function App() {
   return (
     <div className="App">
-      <header className="App-header">Welcome to SquadStack!</header>
+      <header className="App-header">
+        <BrowserRouter>
+          <Routes>
+            <Route path="/" exact element={<Login />} />
+            <Route path="/done" element={<Display />} />
+          </Routes>
+        </BrowserRouter>
```

Comment:
Should router live here? Or should it have a separate place? Where it is easily configurable?